backend/src/main/java/com/google/rolecall/models/CastMember.java

```diff
@@ -45,7 +45,13 @@ public SubCast getSubCast() {
   }
 
   public CastMemberInfo toCastMemberInfo() {
-    return CastMemberInfo.create(id, user.getId(), cast.getId(), order, null);
+    return CastMemberInfo.newBuilder()
+    .setId(id)
+    .setUserId(user.getId())
+    .setSubCastId(cast.getId())
+    .setOrder(order)
+    .setDelete(null)
+    .build();
```

Comment:
There're some formatting issues. 

I strongly encourage use https://github.com/google/google-java-format to format the code.

Reply:
Fixed. I installed the google formatter. Thanks for the recommendation!